### DISCUSSION TITLE
during run python code

### DISCUSSION THREAD

**Discussion by ragewire** [mhammond/pywin32#2521] [General]
I got permission denied during run python code.
"/AppData/Local/Microsoft/WindowsApps/python: Permission denied", what should I do.
![screen](https://user-images.githubusercontent.com/38105030/60216764-85d7d700-9888-11e9-9ae0-5496865e0f8c.png)

**Comment by Avasam** [comment]
@ragewire please provide a minimal reproduction of the code you run to produce your error, and I'll reopen this issue.

It simply looks like you don't have permission to run your Python executable, which isn't related to pywin32.
